[docs][WR-9] Add FAQ page on adding border to text in RN (#12381)
expo/expo: docs/pages/faq/text-border.md

---

> title: Adding a border to text in React Native

Reviewer: ```suggestion
title: Adding a border to text in Expo and React Native apps
```

---

import ImageSpotlight from '~/components/plugins/ImageSpotlight';
import SnackInline from '~/components/plugins/SnackInline';

> To add a border to a `Text` component in React Native, we will need to use a `View` component.

Reviewer: ```suggestion
To add a border to a `Text` component in Expo and React Native apps, we will need to use a `View` component.
```

We will be adding a border to text, similar to the one shown below.

<ImageSpotlight style={{ maxWidth: 276 }} containerStyle={{ marginTop: 0 }} src="/static/images/textborder-example.png" alt="Text rendered with a border" />

> Not what you had in mind? [Reach out to us on the forums.](https://forums.expo.io/)

Reviewer: This prompt reads out of place to me- I lean towards removing it.

## How to add a border to text

> To add a border to text in React Native, wrap the `Text` component with a `View` component. The `View` component has `style` props that allow us to add a border and customize its appearance.

Reviewer: ```suggestion
To add a border to text, wrap the `Text` component with a `View` component. The `View` component has `style` props that allow us to add a border and customize its appearance.
```

> For a full list of `View` `style` props, please refer to the React Native docs on [View Style Props](https://reactnative.dev/docs/view-style-props).

Reviewer: ```suggestion
For a full list of `View` `style` props, please refer to the [View Style Props documentation](../versions/latest/react-native/view-style-props/).
```
